### DISCUSSION TITLE
No puedo instalar jupyter notebook package

### DISCUSSION THREAD

**Discussion by LuceroLuciano** [LaunchX-InnovaccionVirtual/CursoIntroPython#90] [Q&A] [answered]
# Hola explorers. 
Tengo detalles para ejecutar mis notebooks, segun ya instale todo pero aparece que necesito instalar notebook package: 

![Captura de pantalla de 2022-02-10 13-55-31](https://user-images.githubusercontent.com/83784155/153505114-ef61755f-0786-499d-8700-8ea48acdcef7.png)
![Captura de pantalla de 2022-02-10 16-04-15](https://user-images.githubusercontent.com/83784155/153505225-19366031-6516-463d-9681-202a61674219.png)

Le di click en el botón que dice install y también lo instale desde la terminal `/usr/bin/python -m pip install jupyter notebook -U` y me apareció que se instalo correctamente: 
![Captura de pantalla de 2022-02-10 15-13-33](https://user-images.githubusercontent.com/83784155/153505503-0d0a1371-76a7-446b-b6cf-878dc514cdaa.png)
![Captura de pantalla de 2022-02-10 15-13-47](https://user-images.githubusercontent.com/83784155/153505518-0a4635cd-1a22-4980-bff3-907e8858b615.png)

Pero aun así me sigue apareciendo el mensaje: **Running cells with 'Python 3.9.10 64-bit' requires notebook package** y no entiendo porque.

**Comment by MarcosSilvaG** [comment, marked as answer]
Hola!! :D ya intentaste hacer lo que se sugirió en esta otra pregunta?
https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/48

**Reply by LuceroLuciano** [reply]
Si, de hecho volví a instalar el sistema operativo Linux Ubuntu 18.04, pero creo que tendré que probar con Windows porque no me funciona en linux.

**Comment by doguedogue** [comment]
ya reinstalar un sistema operativo para configurar si ya es extremo

El jupyter es extensión del visual code

Mira estos son unos comandos que utilicé en linux y me funcionaron además de seguir la guia paso a paso

```
/usr/bin/python3 -m pip install ipykernel -U --user --force-reinstall
python3 -m pip install ipykernel -U --user --force-reinstall
pip install --upgrade ipython
python -m pip install 'traitlets==4.3.3' --force-reinstall
pip install -U ipykernel
```

**Reply by LuceroLuciano** [reply]
Gracias, lo voy a intentar y les cuento como me fue.

**Reply by LuceroLuciano** [reply]
Muchas gracias por su ayuda explorers. No logre solucionar el detalle con linux, así que decidí trabajar con Windows y ahora todo fluye mejor! :)

**Reply by roblesvargas97** [reply]
Hola compañero me paso lo mismo en linux , trate de resolverlo todo el sábado e inclusive de madrugada y no lo logre así que igual termine haciéndolo en windows
